docs/release-notes.rst

```diff
@@ -10,6 +10,33 @@
  Version 0.29
 **************
 
+Version 0.29.1
+==============
+
+**Release Date:** March 18, 2024
+
+**New Features**
+
+-  Include early-access NVIDIA NGC-based images in our environment offerings. These images are
+   accessible from `pytorch-ngc <https://hub.docker.com/r/determinedai/pytorch-ngc>`__ or
+   `tensorflow-ngc <https://hub.docker.com/r/determinedai/tensorflow-ngc>`__. By downloading and
+   using these images, users acknowledge and agree to the terms and conditions of all third-party
+   software licenses contained within, including the `NVIDIA Deep Learning Container License
+   <https://developer.download.nvidia.com/licenses/NVIDIA_Deep_Learning_Container_License.pdf>`__.
+   Users can build their own images from a specified NGC container version by using the
```

Comment:
Had this before, seems to have gotten lost:
```suggestion
   Users can build their own images from a specified NGC container version using the
```
If you don't like it, that's fine; it doesn't matter much. Just pointing out in case it was unintentional.

Reply:
Thanks, this was unintentional